.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: false
+language: node_js
+node_js:
+  - "0.10"
+before_install:
+  - npm install -g grunt-cli
```

Comment:
might be improved later, but we can use grunt-cli from a local node_module. It's a ++ if we have it on this PR